frontend/common/src/lang/fr.json

```diff
@@ -1093,5 +1093,267 @@
+    "description": "Group for when someone indicates they have a disability"
+  },
+  "RBHmM6": {
+    "defaultMessage": "Autres compétences \"utiles à avoir\" que vous pourriez envisager d'ajouter à votre profil :",
+    "description": "Text that appears when a user is missing optional skills on their profile"
+  },
+  "RQGy+0": {
+    "defaultMessage": "Publiée",
+    "description": "Published pool advertisement status"
+  },
+  "RggAL8": {
+    "defaultMessage": "Modifiez vos options concernant la diversité, l’équité et l’inclusion",
+    "description": "Link text to edit diversity, equity and inclusion information on profile."
+  },
+  "RnSbKI": {
+    "defaultMessage": "Fiabilité ou plus élevée",
+    "description": "Reliability security clearance"
+  },
+  "S9lNLG": {
+    "defaultMessage": "Modifiez vos options d'informations linguistiques",
+    "description": "Link text to edit language information on profile."
+  },
+  "TO/Q6I": {
+    "defaultMessage": "Femme",
+    "description": "Group for when someone indicates they are a woman"
+  },
+  "UCGAzS": {
+    "defaultMessage": "S.O.",
+    "description": "displayed when localized string not available"
+  },
+  "UPx6kk": {
+    "defaultMessage": "<primary>{titre}</primary> à {organization}",
+    "description": "Title at organization"
+  },
+  "X9hYJS": {
+    "defaultMessage": "Fermer le menu"
+  },
+  "ZWR/F3": {
+    "defaultMessage": "Français seulement",
+    "description": "The language requirement is French only."
+  },
+  "b02Azd": {
+    "defaultMessage": "IT-04 (gestionnaire)",
+    "description": "The name of the Senior advisor classification with group and level."
+  },
+  "bOch3v": {
+    "defaultMessage": "Les compétences suivantes \"exigence\" <red>required</red> sont manquantes dans votre profil :",
+    "description": "Text that appears when a user is missing required skills on their profile"
+  },
+  "c39xT8": {
+    "defaultMessage": "Modifier les options de votre expérience.",
+    "description": "Link text to edit experience information on profile."
+  },
+  "cK/hoN": {
+    "defaultMessage": "<primary>{title}</primary> délivré par {issuedBy}",
+    "description": "The award title is issued by some group"
+  },
+  "eFCDP4": {
+    "defaultMessage": "Modifiez vos options de préférences de travail.",
+    "description": "Link text to edit work preferences on profile"
+  },
+  "g2X8Fx": {
+    "defaultMessage": "Fermer la fenêtre de dialogue",
+    "description": "Text for the button to close a modal dialog."
+  },
+  "gfzfP6": {
+    "defaultMessage": "Expirée",
+    "description": "Expired pool advertisement status"
+  },
+  "gz2VrQ": {
+    "defaultMessage": "Intermédiaire",
+    "description": "Intermediate, skill level"
+  },
+  "hDupu9": {
+    "defaultMessage": "Modifiez vos options de compétences et d'expérience.",
+    "description": "Link text for editing skills and experiences on profile."
+  },
+  "jp+WTk": {
+    "defaultMessage": "Ouvrir le menu"
```

Comment:
Is it possible to add a description to the English string so that one can find the English string if necessary?